### ISSUE TITLE
How to install module

### ISSUE THREAD

**Issue by Twissall** [divyangbw/ngBaasbox#7]
Hi. 

Pardon me if this is a noob question. But could you please give an example of how to install this module into an angular app. 

Many thanks.

**Comment by divyang-joshi** [comment]
Hey. Apologies for such a late response. So I'm working on a few things that might change the plugin itself to make it more adaptive for future baasbox releases. Having said that, it shouldn't change how you'd use this module. If anything changes, I'll update the readme :)

To answer your question (excuse the simplicity. Just making sure others are taken care of):
1. Download the module by either downloading the whole thing, forking, or just opening the file (ngBaasbox.js) under src.
2. Add this file to your project. For example if you have a lib folder or something similar. 
3. In your index.html, make a include the file.
4. Finally, to include this in your angular project, in your app.js (main entry point) file:

```
// ngBaasbox is what we need to include
var app = angular.module('app', ['ngBaasbox', 'app.controllers', 'app.services', 'app.factory'])
```
1. To use it, lets take an controller file example:

```
// Notice how I included $baasbox here
angular.module('app.controllers')
    .controller('HomeCtrl', function ($scope, $baasbox) {

        // Create a new account
        function signup(userToRegister) {
            var q = $q.defer();
            $baasbox.signup(userToRegister).then(function (user) {
                // Success
                q.resolve(user)
            }, function (err) {
                // Fail
                q.reject(err)
            });
            return q.promise;
        }
    }
```

So in short, include the `ngBaasbox` in app.js and wherever you want to use it, simply include it as `$baasbox`. This can be controllers, services, anything really. If you have any more questions, feel free to ask. If something isn't clear, ask :)
